everett/manager.py

```diff
@@ -237,12 +313,15 @@ class ConfigIniEnv(object):
     """
     def __init__(self, possible_paths):
         self._parser = None
+        if isinstance(possible_paths, six.string_types):
+            possible_paths = [possible_paths]
```

Comment:
I like this. Further we can pull this out and create `listify` or something like that:

```
def listify(thing):
    if isinstance(thing, six.string_types):
        return [thing]
    return thing


possible_paths = listify(possible_paths)
```

That lets us use it for namespaces and other things that are now `string|list-of-strings`.

I'll do that in a follow-up PR along with doing a pass on the docs to add examples for the single-string case.